src/components/ProgressBars/LinearProgressBar/LinearProgressBar.tsx

```diff
@@ -163,7 +187,10 @@ const LinearProgressBar: VibeComponent<LinearProgressBarProps, HTMLDivElement> &
 Object.assign(LinearProgressBar, {
   styles: ProgressBarStyle,
   barStyles: ProgressBarStyle,
-  sizes: SIZES
+  types: ProgressBarType,
+  barTypes: ProgressBarType,
+  sizes: SIZES,
+  defaultTestId: ComponentDefaultTestId.LINEAR_PROGRESS_BAR
```

Comment:
I don't think this is a good practice. this could lead to duplicated non-unique selectors in the code